### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by legogoUA**

### CODE REVIEW

File: src/makeCalculator.js
Comment (VitaliyBodnar) [inline src/makeCalculator.js:66]: - it's better to return `this` instead of `calculator` as it will make a code more scalable 
- only reset and operate should return this, other method should't. Check again the example.

Screenshot 2023-10-24 at 17 34 30

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/makeCalculator.js
Comment (maxim2310) [inline src/makeCalculator.js:53]: need to add check if num === 0

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just one tiny thing needs to be fixed before approval)

File: src/makeCalculator.js
Comment (etojeDenys) [inline src/makeCalculator.js:45]: actually, it's redundant +0, it doesn't affect the calculator at all

File: src/makeCalculator.js
Comment (etojeDenys) [inline src/makeCalculator.js:53]: same here

File: src/makeCalculator.js
Comment (etojeDenys) [inline src/makeCalculator.js:62]: same here

File: src/makeCalculator.js
Comment (etojeDenys) [reply src/makeCalculator.js:79]: still not fixed comments

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
